### DISCUSSION TITLE
使用pythonAPI接口时的问题

### DISCUSSION THREAD

**Discussion by ghost** [MaaAssistantArknights/MaaAssistantArknights#7931] [Q&A]
![11111111111111](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/111691569/98742663-0615-4605-ac51-40350db45fb5)

就像是这样
报了两次libpng warning: sBIT: invalid的错误之后
再过一段时间之后程序会结束而模拟器无任何反应
不知道是哪出问题了

**Comment by ghost** [comment]
呃，我似乎找不到 https://github.com/orgs/MaaAssistantArknights/discussions/7931 的重复项在哪
[https://opengraph.githubassets.com/60f2ae16cdf291a79d20d659142e9d15fb300e915b8821436f80ba9bdafddc3c/orgs/MaaAssistantArknights/discussions/7931]<https://github.com/orgs/MaaAssistantArknights/discussions/7931>
使用pythonAPI接口时的问题 ・ MaaAssistantArknights ・ Discussion #7931<https://github.com/orgs/MaaAssistantArknights/discussions/7931>
就像是这样 报了两次libpng warning: sBIT: invalid的错误之后 再过一段时间之后程序会结束而模拟器无任何反应 不知道是哪出问题了
github.com

________________________________
发件人: status102 ***@***.***>
发送时间: 2024年1月6日 19:21
收件人: MaaAssistantArknights/MaaAssistantArknights ***@***.***>
抄送: 雷石东 ***@***.***>; Author ***@***.***>
主题: Re: [MAA] 使用pythonAPI接口时的问题 (Discussion #7931)


Closed #7931<https://github.com/orgs/MaaAssistantArknights/discussions/7931> as duplicate.

―

**Reply by status102** [reply]
Duplicate of #7933
